spec/javascripts/components/step-by-step-nav-spec.js

```diff
@@ -446,17 +446,19 @@ describe('A stepnav module', function () {
 
     it('opens the steps it has remembered', function () {
       var $step1 = $element.find('#topic-step-one')
-      expect($step1).toHaveClass('step-is-shown')
-      expect($step1.find('.js-panel')).not.toHaveClass('js-hidden')
+      expect(window.isClassOnElement($step1[0], 'step-is-shown')).toEqual(true)
+      var $jsPanel = $step1.find('.js-panel')
+      expect(window.isClassOnElement($jsPanel[0], 'js-hidden')).toEqual(false)
```

Comment:
Totally non-blocking suggestion here, but I think this could be tweaked to reduce the reliance on jQuery's `find` function.

```Javascript
var $jsPanel = $step1[0].querySelector('.js-panel')
expect(window.isClassOnElement($jsPanel, 'js-hidden')).toEqual(false)
```

Reply:
Thanks @andysellick - yeah I guess as our priority was removing `jasmine-jquery` instead of jQuery itself we were being cautious about only removing jQuery when we had to. I can update it so it uses vanilla JS though.

Reply:
@andysellick I think it's better we tackle this if we look at removing jQuery from the tests in the future, as it's a bit unrelated to this PR (hence why you said it's non-blocking.) Thanks for raising it though!

Reply:
Fair enough!